content/docs/atplatform/specification.md

```diff
@@ -146,7 +146,7 @@ Metadata of a key should describe the following properties of the value being in
 | ttb | No | Time to birth in milliseconds. |
 | availableFrom | Yes | A Date and Time derived from the ttl (now + ttl). A Key should be only available after availableFrom |
 | isCached | No | True if the key is cached |
-| ttr | No | Time in milliseconds after which the cached key needs to be refreshed. Ttr of -1 indicates that the key can be cached forever. |
+| ttr | No | Time in milliseconds after which the cached key needs to be refreshed. ttr of null indicates non-cacheable. ttr of -1 indicates that the key can be cached forever. ttr of 0 indicates do not refresh. ttr of > 0 will refresh the key. |
```

Comment:
Is null treated the same as 0 ?

Reply:
i mentioned they have the same effect (do not cache) in this commit:
https://github.com/atsign-foundation/docs.atsign.com/pull/247/commits/eba0d8250569096d5434e7b208b366ed9ef4e590